package.json

```diff
@@ -187,7 +187,7 @@
         "vscode-languageclient": "^6.1.3"
     },
     "scripts": {
-        "clean": "rimraf out && rimraf lib && rimraf vscode-lingua-franca-*.vsix",
+        "clean": "rimraf out && rimraf lib && rimraf lingua-franca/lsp/build && rimraf vscode-lingua-franca-*.vsix",
```

Comment:
I think we should call clean whenever we run `compile`, no?

Reply:
`compile` is just for the JavaScript/Typescript code and does not involve rebuilding the JAR. The supported build process for building from source is `npm install`, so I think it is sufficient to ensure that that command works without worrying about what will happen if users try to run other commands when their file system is in a specific state.